coffee/chosen.jquery.coffee

```diff
@@ -105,6 +106,18 @@ class Chosen extends AbstractChosen
     else
       @container.on 'click.chosen', (evt) -> evt.preventDefault(); return # gobble click of anchor
 
+      @focus_field.on 'blur.chosen', (evt) => this.input_blur(evt); return
+      @focus_field.on 'focus.chosen', (evt) => this.input_focus(evt); return
+
+      transfer_value = () =>
+        @search_field.val(@search_field.val() + @focus_field.val())
```

Comment:
I left this concatenation in because @adunkman said there's a way to leave a single select and come back to it with a value present. I haven't been able to reproduce that, but I trust @adunkman.

Reply:
You may be trusting me more than I trust me… I’ll do some extra testing and try to figure it out, otherwise I’ll add a commit to drop the concat. 🐈

Reply:
Couldn’t figure it out! Must have been mistaken. Removed the concat and squashed the commits to prep for final review.